src/engine/UIManager.cxx

```diff
@@ -976,3 +991,11 @@ void UIManager::changeFullScreenMode(UIElement *sender)
     Engine::instance().map->refresh();
   }
 }
+
+void UIManager::changeMusicVolume(UIElement* sender) 
+{
+  // TODO: Save settings
```

Comment:
Is this TODO relevant?

Reply:
Not anymore, you're right. Thanks for catching this!